common/common.podspec

```diff
@@ -0,0 +1,44 @@
+Pod::Spec.new do |spec|
+    spec.name                     = 'common'
+    spec.version                  = '1.0.0'
+    spec.homepage                 = 'https://github.com/jan-gogo/run_blocking'
+    spec.source                   = { :git => "Not Published", :tag => "Cocoapods/#{spec.name}/#{spec.version}" }
+    spec.authors                  = ''
+    spec.license                  = ''
+    spec.summary                  = 'Common module for run blocking example'
+
+    spec.static_framework         = true
+    spec.vendored_frameworks      = "build/cocoapods/framework/#{spec.name}.framework"
+    spec.libraries                = "c++"
+    spec.module_name              = "#{spec.name}_umbrella"
+
+            
+
+    spec.pod_target_xcconfig = {
+        'KOTLIN_TARGET[sdk=iphonesimulator*]' => 'ios_x64',
+        'KOTLIN_TARGET[sdk=iphoneos*]' => 'ios_arm',
+        'KOTLIN_TARGET[sdk=watchsimulator*]' => 'watchos_x86',
+        'KOTLIN_TARGET[sdk=watchos*]' => 'watchos_arm',
+        'KOTLIN_TARGET[sdk=appletvsimulator*]' => 'tvos_x64',
+        'KOTLIN_TARGET[sdk=appletvos*]' => 'tvos_arm64',
+        'KOTLIN_TARGET[sdk=macosx*]' => 'macos_x64'
+    }
+
+    spec.script_phases = [
+        {
+            :name => 'Build common',
+            :execution_position => :before_compile,
+            :shell_path => '/bin/sh',
+            :script => <<-SCRIPT
+                set -ev
+                REPO_ROOT="$PODS_TARGET_SRCROOT"
+                "$REPO_ROOT/../gradlew" -p "$REPO_ROOT" :common:syncFramework \
```

Comment:
We intentionally don't use Cocoapods because it's slow and introduces a Ruby dependency. Here's an example of accomplishing the same task with an Xcode shell script:

- https://github.com/russhwolf/multiplatform-hello/blob/d880b3579895c1cf11fb291624113829426affe7/ios/Multiplatform%20Hello.xcodeproj/project.pbxproj#L301

Reply:
Thanks for review, I'll implement fix ASAP.